experimenter/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormFeatureValue/validators.ts

```diff
@@ -332,3 +333,30 @@ export function schemaAutocomplete(schema: Record<string, unknown>) {
     return null;
   };
 }
+
+/**
+ * Parse from FML errors to Diagnostics
+ */
+function parseFmlErrors(
+  view: TestableEditorView,
+): Diagnostic[] {
+  const diagnostics: Diagnostic[] = [];
+  const line = 0;
+  const column = 0;
+
+  diagnostics.push({
+    from: line,
+    to: column + "error".length ?? 0,
+    message: "oh no!",
+    severity: "error"
+  });  
+  
+  return diagnostics;
+}
+
+export function fmlLinter(schema: Record<string, unknown>) {
```

Comment:
fml linter shouldnt be dependant on a schema. Its based on the feature config + application channel.